envoy/Makefile

```diff
@@ -25,6 +25,7 @@ ENVOY_BINS = \
 CHECK_FORMAT ?= ./bazel-bin/check_format.py.runfiles/envoy/tools/check_format.py
 
 BAZEL ?= $(QUIET) bazel
+BAZEL_FILTER ?= 2>&1 | grep -v -e "bazel-out/.*/genfiles/external/.*: warning: directory does not exist."
```

Comment:
won't this hide the result of bazel running, and instead report the success or failure of the `grep`?

Reply:
You're right.